packages/vitest/src/node/reporters/renderers/utils.ts

```diff
@@ -223,9 +223,9 @@ export function formatProjectName(name: string | undefined, suffix = ' ') {
     .split('')
     .reduce((acc, v, idx) => acc + v.charCodeAt(0) + idx, 0)
 
-  const colors = [c.black, c.yellow, c.cyan, c.green, c.magenta]
+  const colors = [c.bgYellow, c.bgCyan, c.bgGreen, c.bgMagenta]
```

Comment:
colors are also specified in the UI package, they need to follow the same pattern

Reply:
By "the same pattern" do you mean that the UI package should not use `c.inverse`? If so, `c.inverse` is used in many other places as well, but (as far as I can tell) in ways that seem less likely to produce unreadable text than this case.

Reply:
I think it just needs to match when users look at both terminal and UI, but not exact color nor style. The code is probably this one:

https://github.com/vitest-dev/vitest/blob/c82387d36f7dc0cf8f558b96bf97f782b0b3637a/packages/ui/client/utils/task.ts#L21-L30

Reply:
@hi-ogawa Thanks for that pointer! I wasn't aware of this. The colors are consistent now as of commit 267863d. 🙂